src/main/java/com/google/sps/data/LoginObject.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+public class LoginObject {
+  String id;
```

Comment:
While combining id & error messages (as you do below) can be more efficient with variables, it's semantically unclear for client that handles your api. For that reason it's almost always better to have a dedicated "errorMessage" variable or something like that.

Reply:
Added an error message attribute to LoginObject